crates/subspace-farmer/src/bin/subspace-farmer/utils.rs

```diff
@@ -1,5 +1,28 @@
+use prometheus_client::metrics::histogram::{exponential_buckets, Histogram};
+use prometheus_client::registry::{Registry, Unit};
+use subspace_farmer_components::auditing::AuditEvent;
 use tokio::signal;
 
+#[derive(Debug, Clone)]
+pub(crate) struct FarmerMetrics {
+    audit: Histogram,
+}
+
+impl FarmerMetrics {
+    pub(crate) fn new(registry: &mut Registry) -> Self {
+        let sub_registry = registry.sub_registry_with_prefix("farmer");
+
+        let audit = Histogram::new(exponential_buckets(0.001, 2.0, 12));
```

Comment:
We will need a label with farm ID or else it will not be possible to distinguish audit times of different farms that could be on SSDs with different performance characteristics.